pkgs/applications/misc/kupfer/default.nix

```diff
@@ -14,12 +14,14 @@
 with python3Packages;
 
 buildPythonApplication rec {
-  name = "kupfer-${version}";
+  pname = "kupfer";
   version = "319";
 
-  src = fetchurl {
-    url = "https://github.com/kupferlauncher/kupfer/releases/download/v${version}/kupfer-v${version}.tar.xz";
```

Comment:
This is a dist tarball, not a repo snapshot. Will this not break the application version information?

Reply:
You are correct. My mistake. Thank you for catching.